packages/api/src/graphql/mutation.private.ts

```diff
@@ -737,6 +740,32 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
         authorise(CanPublishPage, roles)
         return dbAdapter.page.unpublishPage({id})
       }
+    },
+
+    takeActionOnComment: {
```

Comment:
I read in an article that this is considered bad practice. Every mutation should have one job and i agree with that philosophy. I suggest you create two mutations called `approveComment` and `rejectComment`. The database adapter method on the other hand should be simple and can be called `updateComment`.

Reply:
Agree, We currently have three mutations to approve, reject or request changes from the user.